rs/consensus/src/consensus/notary.rs

```diff
@@ -49,7 +49,11 @@ use std::{sync::Arc, time::Duration};
 /// The acceptable gap between the finalized height and the certified height. If
 /// the actual gap is greater than this, consensus starts slowing down the block
 /// rate.
-const ACCEPTABLE_FINALIZATION_CERTIFICATION_GAP: u64 = 3;
+const ACCEPTABLE_FINALIZATION_CERTIFICATION_GAP: u64 = 2;
+
+/// The amount of time consensus should delay notarization of the next block by,
+/// for each height that the latest finalized block is ahead of the latest certified state.
+const BACKLOG_DELAY_MILLIS: u64 = 2000;
```

Comment:
Should we explain here how this value was chosen?

Reply:
I was trying to keep the slope somewhat similar to what we had before (1s & 3s) to be cautious. Empirically, 2s also worked better then 1s, but this is something we can adjust further if needed. I updated the comment